core/kubernetes/openhim/openhim-mongo-volume-claim.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: openhim-mongo-volume-claim
+spec:
+  storageClassName: "standard"
```

Comment:
Are the quotes needed for this property?

Reply:
I've removed this field. Wasn't needed